localstack-core/localstack/services/cloudwatch/provider.py

```diff
@@ -364,6 +364,8 @@ def get_raw_metrics(self, request: Request):
                 "v": r.value,
                 "t": r.timestamp,
                 "d": [{"n": d.name, "v": d.value} for d in r.dimensions],
+                "account": account_id,
+                "region": region,
```

Comment:
q: How is this change related to the OpenAPI specs?

Reply:
This makes the older CloudWatch provider return the same API response as the v2. I agree this can go in a separate PR. I will raise is separately.

Reply:
Honestly, I am not sure this is worth the effort. The endpoint is currently not even documented, it was introduced a long time ago as a [customer requested it](https://github.com/localstack/localstack/issues/3508). But back then this endpoint could be used by simply using the browser. 

With the requirement change of having a valid request-header where we could extract the account and region, IMO this endpoint become unusable (I think this was introduced with this [PR](https://github.com/localstack/localstack/pull/5478)). 

For v2 we removed this requirement, e.g. it's possible to just browse the page without any request headers. This is now feasible as we store all the data in sqlite and simply dump the entire database (also returning the account and region fields).